packages/amplication-client/src/Workspaces/WorkspaceLayout.tsx

```diff
@@ -1,102 +1,57 @@
-import React from "react";
-import { match, Switch } from "react-router-dom";
+import React, { lazy } from "react";
+import { match } from "react-router-dom";
 import MainLayout from "../Layout/MainLayout";
-import MenuItemWithFixedPanel from "../Layout/MenuItemWithFixedPanel";
-import ScreenResolutionMessage from "../Layout/ScreenResolutionMessage";
+
+// import MenuItemWithFixedPanel from "../Layout/MenuItemWithFixedPanel";
```

Comment:
Can you please explain the reason we push this commented code? I'd rather not push commented lines of code in any case. If we need to look back at it and grab it, we can always check the Git history and do that.